tests/integration/update.bats

```diff
@@ -123,6 +124,34 @@ function check_cgroup_value() {
         check_cgroup_value $CGROUP_MEMORY "memory.memsw.limit_in_bytes" 96468992
     fi
 
+    if [ -f "$CGROUP_MEMORY/memory.memsw.limit_in_bytes" ]; then
```

Comment:
For casting -1 to uint64 I think that's a defined behavior and pretty explanatory, but since you missed it on the first try it says that can be more intuitive. being said that I'll also add a check explicitly for -1.